src/vs/code/electron-main/auth.ts

```diff
@@ -60,10 +60,12 @@ export class ProxyAuthHandler extends Disposable {
 			title: 'VS Code',
 			webPreferences: {
 				preload: URI.parse(require.toUrl('vs/base/parts/sandbox/electron-browser/preload.js')).fsPath,
-				enableWebSQL: false,
 				sandbox: true,
-				devTools: false,
+				contextIsolation: true,
+				enableWebSQL: false,
 				enableRemoteModule: false,
+				nativeWindowOpen: true,
```

Comment:
`nativeWindowOpen` is not needed when `sandbox: true`, since that is the default behavior in this mode.

Reply:
Ok pushed a change to remove it.